mkdocs.yml

```
@@ -0,0 +1,7 @@
+site_name: E-file Proxy Server
```

```suggestion
site_name: SuffolkLITLab E-file Proxy Server
```

Not sure what the branding is on this, but it does make the source a bit more clear. Take it or leave it!

---

Good point, but I ended up using `Suffolk LIT Lab` instead.